Cargo.toml

```diff
@@ -41,6 +41,7 @@ rand = "0.8.3"
 trybuild = "1.0.23"
 serde = { version = "1.0.117", features = ["derive"] }
 serde_test = "1.0.117"
+serde_json = "1.0.117"
```

Comment:
JSON is a poor choice to illustrate column-major serialization. If you want the output to be human-readable, use row-major serialization. If you don't, don't use a slow, complex, text-based format.

Reply:
Okay, should I use bincode instead or something else?

Reply:
If you want to illustrate columnar serialization, bincode would be a reasonable choice.